torch/fx/experimental/symbolic_shapes.py

```diff
@@ -5506,7 +5535,13 @@ def resimplify_floor_div(axioms: Dict[sympy.Expr, sympy.Expr]) -> None:
             var_ranges = dict(var_to_range)
 
         symbol_info = tuple(
-            (s, var_ranges.get(s), self.var_to_val.get(s), s in self.size_like)
+            (
+                s,
+                var_ranges.get(s),
+                self.var_to_val.get(s),
+                s in self.size_like,
+                self.oblivious_upper_bound_exclusive.get(s),
+            )
             for s in sorted(fs, key=lambda s: str(s))  # TODO: speed up sort?
```

Comment:
```suggestion
            for s in sorted(fs, key=str)  # TODO: speed up sort?
```
This will marginally speedup sort btw

Reply:
https://github.com/pytorch/pytorch/pull/144743